src/python/evaluation/evaluation_config.py

```diff
@@ -0,0 +1,44 @@
+import logging.config
+from pathlib import Path
+from typing import List, Union
+
+from src.python.common.tool_arguments import RunToolArguments
+from src.python.evaluation.common.util import EvaluationProcessNames
+from src.python.review.application_config import LanguageVersion
+from src.python.review.common.file_system import create_directory
+
+logger = logging.getLogger(__name__)
+
+
+class EvaluationConfig:
+    def __init__(self, args):
+        self.tool_path: Union[str, Path] = args.tool_path
+        self.output_format: str = args.format
+        self.xlsx_file_path: Union[str, Path] = args.xlsx_file_path
+        self.traceback: bool = args.traceback
+        self.output_folder_path: Union[str, Path] = args.output_folder_path
+        self.output_file_name: str = args.output_file_name
+
+    def build_command(self, inspected_file_path: Union[str, Path], lang: str,
+                      run_tool_arguments=RunToolArguments) -> List[str]:
```

Comment:
I think you should remove run_tool_arguments from function arguments. You always use RunToolArguments. FORMAT.value.short_name